ActindoWorkFlow/Operation/01_ManageWorkflows.md

```diff
@@ -0,0 +1,423 @@
+
+  ![Search action](/Assets/Screenshots/ActindoWorkFlow/Workflows/SearchAction.png "[Search action]")
+
+2. Click the *search for an action* field and enter the name of the transition or a keyword you are searching for. If you have clicked the ![Add](/Assets/Icons/Plus04.png "[Add]") (Add) button next to a place, only those transitions that are compatible with the data type of the place are displayed in the list.
+
+  > [Info] The list of transitions is filtered for your keyword as you type.
+
+3. Click the transition you want to add in the list of transitions.
+  The transition is added to the workflow. If the transition was added from a place which requires a certain input data type, the arc from the place is automatically connected to the corresponding input port of the transition.
+
+  ![First action](/Assets/Screenshots/ActindoWorkFlow/Workflows/FirstAction.png "[First action]")
+
+  > [Info] If any input port or output port of the new transition is mandatory, an arc and a place are automatically displayed with the corresponding transition.
+
+
+#### Connect the transition
+
+1. Click the ![Add](/Assets/Icons/Plus04.png "[Add]") (Add) button next to an output port of the transition.    
+  A new place connected with an arc to the selected output port is displayed.
+
+  ![First place](/Assets/Screenshots/ActindoWorkFlow/Workflows/FirstPlace.png "[First place]")
+
+2. Add further transitions by following the procedure [Create an action](#create-an-action) or connect the place with the end place as described in the following step.
+
+  > [Info] A workflow must contain at least one transition and can contain an unlimited number of transitions.
+
+3. Click the place, drag it over the place to which you want to connect it and drop it.     
+  The places are merged and the new connection is created.
+
+
+### Next steps
+
+- [Edit a workflow](#edit-a-workflow)
+- [Copy a workflow](#copy-a-workflow)
+- [Publish a workflow version](#publish-a-workflow-version)
+- [Unpublish a workflow version](#unpublish-a-workflow-version)
+- [Track a workflow process](02_TrackProcesses.md)
+
+### See also
+
+- [User Interface workflows](/ActindoWorkFlow/UserInterface/00_UserInterface.md)
+- [Workflow and process elements](/ActindoWorkFlow/Overview/02_WorkflowProcessElements.md)
+
+
+
+## Edit a workflow
+
+Edit a workflow to modify elements within the workflow, to extend the workflow by adding further transitions or to reduce the workflow by removing transitions.
+
+### Prerequisites
+
+- At least one workflow is created, see [Create a workflow](#create-a-workflow).
+- You have the required rights to edit a workflow.
+
+### Procedure
+
+*Workflows > Workflows > Tab OVERVIEW*
+
+![Workflows](/Assets/Screenshots/ActindoWorkFlow/Workflows/Workflows.png "[Workflows]")
+
+1. Click the workflow you want to edit in the list of workflows.   
+  The *Workflow Versions* view with all versions of the selected workflow is displayed.
```

Comment:
```suggestion
  The *Workflow Versions* view with all versions of the selected workflow is displayed.
```The *Workflow versions*...

Wieder das Thema mit groß/klein. Ob es sich lohnt bzw. ob es zeitlich zu schaffen ist, für diesen Release das zu vereinheitlichen...